dev/docker-compose.yaml

```diff
@@ -24,7 +24,7 @@ x-airflow-common:
     OPENLINEAGE_URL: http://host.docker.internal:5050/
     OPENLINEAGE_EXTRACTORS: "astronomer.providers.google.cloud.extractors.bigquery.BigQueryAsyncExtractor;\
                             astronomer.providers.snowflake.extractors.snowflake.SnowflakeAsyncExtractor;\
-                            astronomer.providers.amazon.aws.extractors.redshift.RedshiftAsyncDataExtractor"
+                            astronomer.providers.amazon.aws.extractors.redshift.RedshiftAsyncExtractor"
```

Comment:
Interesting how it was tested during development with the wrong extractor path.

cc @rajaths010494

Reply:
I believe that is the case @pankajkoti